docs/janssen-server/install/docker-install/compose.md

```diff
@@ -1,147 +1,111 @@
 ---
-tags:
-- administration
-- installation
-- quick-start
-- docker compose
-- docker image
+title: Docker Compose
 ---
 
-# Docker compose
+# Installing and Running Janssen with Docker Compose
 
+This document provides instructions for running Janssen using Docker Compose. It's designed as a flexible deployment tool for development, testing, and light proof-of-concept use cases where a straightforward deployment is needed on a single machine (single node deployment) either on-premises or in the cloud. For production environments or for high availability and scalability, a Kubernetes-based deployment is recommended.
 
-> **Warning**
-> This image is for testing and development purposes only. Use Janssen [helm charts](https://github.com/JanssenProject/jans/tree/main/charts/janssen) for production setups.
+## Prerequisites
 
+- Install [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/).
+- Depending on your operating system and the architecture of your CPU, you may need to build local images. See [building local images](#building-custom-images-optional)
 
+## System Requirements
 
-Docker monolith image packaging for Janssen. This image packs janssen services including the auth-server, config-api, fido2, and scim.
+The following system requirements are as per the single mode deployment:
 
-## Pre-requisites
+| Component  | CPU Unit | RAM  | Disk Space                |
+|------------|----------|------|---------------------------|
```

Comment:
@saikumarvasa100-hash We do not want to prescribe system requirements for development environments. Please remove this section.